samples/tokenCredentialSamples/AuthenticationCodeFlow/src/index.ts

```diff
@@ -0,0 +1,126 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+
+/**
+ * This sample is referenced from - https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/identity/identity/samples/authorizationCodeSample.ts
+ */
+import { AuthorizationCodeCredential } from "@azure/identity";
+import express from "express";
+import { Server } from "http";
+import "isomorphic-fetch";
+import open from "open";
+import qs from "qs";
+
+import { TokenCredentialAuthenticationProvider } from "../../../../authProviders/azureTokenCredentials";
```

Comment:
should we just import our own sdk instead so the sample can be standalone from the source.

Reply:
Done. Added the local file path as a dependency.